src/clr-addons/quick-list/add-option/add-option.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="quick-list-input">
+  <clr-select-container>
+    <select class="w-100" clrSelect name="options" [(ngModel)]="selectedValue" (ngModelChange)="selectedValue = $event">
```

Comment:
get rid of the (ngModelChange) - you already have 2 way binding there.

why is there a w-100? selects should have 100% width automatically.

Reply:
removing the w-100 makes the select panel only cover the largest available label

Reply:
I see yes you are right, our custom CSS which sets selects to 100% width only works inside forms, you are not using forms inside this component

Reply:
so, I leave it now